fabric-fluid-v1/src/main/java/net/fabricmc/fabric/api/fluids/v1/container/volume/FluidVolumeDelegate.java

```diff
@@ -0,0 +1,33 @@
+package net.fabricmc.fabric.api.fluids.v1.container.volume;
+
+import net.minecraft.fluid.Fluid;
+import net.minecraft.nbt.CompoundTag;
+
+import net.fabricmc.fabric.Action;
+
+public class FluidVolumeDelegate implements FluidVolume {
```

Comment:
Personally I'd call it a `ForwardingFluidVolume` instead.

Reply:
intellij calls it a delegate